docs/content/preview/admin/yb-admin.md

```diff
@@ -615,25 +615,23 @@ Flush the memstores of the specified table on all tablet servers to disk.
 ```sh
 yb-admin \
     -master_addresses <master-addresses> \
-    flush_table <table_name> | <table_id> <db_type>.<namespace> [timeout_in_seconds] [ADD_INDEXES]
+    flush_table <table> | <db_type>.<namespace> [timeout_in_seconds] [ADD_INDEXES]
```

Comment:
This is incorrect.  It doesn't even match the example on line 635,633.  The official yb-admin help output would reveal (expanded out)

`flush_table ( [(ycql|ysql).]<namespace_name> <table_name> | tableid.<table_id> ) ...`

The original help shows:

```
 20. flush_table <table> [<timeout_in_seconds>] (default 20) [ADD_INDEXES] (default false)
 21. flush_table_by_id <table_id> [<timeout_in_seconds>] (default 20) [ADD_INDEXES] (default false)
...
<namespace>:
 [(ycql|ysql).]<namespace_name> (default ycql.)
<table>:
 <namespace> <table_name> | tableid.<table_id>
<index>:
  <namespace> <index_name> | tableid.<index_id>
```

Reply:
Thanks @jasonyb. Is the following 2 ways of using flush_table correct? And should the second one be flush_table_by_id instead? 
1. `yb-admin \
    -master_addresses <master-addresses> \
    flush_table <db_type>.<namespace> <table> [<timeout_in_seconds>] [ADD_INDEXES]`

* *master_addresses*: Comma-separated list of YB-Master hosts and ports. Default value is `localhost:7100`.
* *db_type*: The type of database. Valid values include ysql, ycql.
* *namespace*: The name of the database (for YSQL) or keyspace (for YCQL).
* *table*: The name of the table to flush.
* *timeout_in_seconds*: Specifies duration, in seconds when the CLI timeouts waiting for flushing to end. Default value is `20`.
* *ADD_INDEXES*: If the database should also flush the secondary indexes associated with the table. Default is `false`.

2. `yb-admin \
    -master_addresses <master-addresses> \
    flush_table tableid.<table_id> [<timeout_in_seconds>] [ADD_INDEXES]`

* *master_addresses*: Comma-separated list of YB-Master hosts and ports. Default value is `localhost:7100`
* *table*: The name of the table to flush.
* *table_id*: The identifier (ID) of the table.
* *timeout_in_seconds*: Specifies duration, in seconds when the CLI timeouts waiting for flushing to end. Default value is `20`.
* *ADD_INDEXES*: If the database should also flush the secondary indexes associated with the table. Default is `false`.

Reply:
Those two are good, and `flush_table_by_id` would be a third way, but since it is duplicate of `flush_table tableid.<table_id>`, I don't think you need to document it.  We may even deprecate it in the future.